contracts/script/SetUpEigenDA.s.sol

```diff
@@ -111,9 +128,16 @@ contract SetupEigenDA is EigenDADeployer, EigenLayerUtils {
             operatorETHAmounts[i] = 5 ether;
         }
 
+        uint256 clientPrivateKey = 0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcded;
```

Comment:
maybe we could use one of the accounts prefunded in anvil?

Reply:
I realized that I could send deposits to the client as the owner and there's no need to access client private key here, just the target address:-)